luajit: bump new version (#11693)
tarantool/tarantool: changelogs/unreleased/gh-11229-sysprof-report.md

## feature/tools

> LuaJIT's platform profile function `misc.sysprof.report()` may be called
> during the profiling as well (gh-11229).

Reviewer: ```suggestion
It is now possible to call LuaJIT's platform profile function `misc.sysprof.report()`
during the profiling as well (gh-11229).
```
